Docs/source/highlights.rst

```diff
@@ -185,3 +195,21 @@ Nuclear Fusion - Magnetically Confined Plasmas
    **Accelerated kinetic model for global macro stability studies of high-beta fusion reactors**.
    Physics of Plasmas **30**, 122508, 2023.
    `DOI:10.1063/5.0178288 <https://doi.org/10.1063/5.0178288>`__
+
+Plasma Thrusters and Electric Propulsion
+****************************************
+
+#. Xie L., Luo X., Zhou Z. and Zhao Y.,
+   **Effect of plasma initialization on 3D PIC simulation of Hall thruster azimuthal instability**.
+   Physica Scripta, **99**, 095602, 2024.
+   `DOI:10.1088/1402-4896/ad69e5 <https://doi.org/10.1088/1402-4896/ad69e5>`__
+
+#. Marks T. A. and Gorodetsky A. A.,
+   **Hall thruster simulations in WarpX**.
+   38th International Electric Propulsion Conference, Toulouse, France, 2024.
+   `DOI:10.7302/234915 <https://doi.org/10.7302/23491>`__
+
+#. Boeuf J-P, Smolyakov A.
+   **Physics and instabilities of low-temperature E × B plasmas for spacecraft propulsion and other applications**.
+   Physics of Plasmas **30**, 050901, 2023.
+   `DOI:10.1063/5.0145536 <https://doi.org/10.1063/5.0145536>`__
```

Comment:
```suggestion
```

Reply:
Correct, just aspirational in the end :+1: